helm/messenger/conf/nginx.conf

```diff
@@ -80,9 +80,10 @@ server {
     try_files    $uri $uri/ /index.html;
   }
 
-  location = /conf.toml {
+  location = /conf {
     root       /var/www;
     expires    1m;
+    try_files  /$host.yaml /$host.toml $uri.toml $uri.yaml $uri  =404;
   }
```

Comment:
Since configuration is no longer bound to a single format, I've dropped the `.toml` from location. So it'll be just `localhost/conf` instead of `localhost/conf.toml`. Nginx will try to find `.yaml` and `.toml` files in the directory.